src/com/carolinarollergirls/scoreboard/core/impl/TeamImpl.java

```diff
@@ -85,15 +85,15 @@ protected Object computeValue(PermanentProperty prop, Object value, Object last,
             Timeout t = scoreBoard.getCurrentTimeout(); 
             return t.isRunning() && this == t.getOwner() && t.isReview();
         }
-        if (prop == Value.TRIP_SCORE && flag != Flag.COPY && scoreBoard.isInJam() && (Integer)value > 0) {
-            
+        if (prop == Value.TRIP_SCORE && flag != Flag.COPY && (Integer)value > 0) {
             // If points arrive during an initial trip and we are not in overtime, assign the points to the first scoring trip instead.
             if (getCurrentTrip().getNumber() == 1 && !getScoreBoard().isInOvertime()) {
                 set(Value.TRIP_SCORE,0);
                 execute(Command.ADD_TRIP);
                 set(Value.TRIP_SCORE,value);
             }
-
+        }
+        if (prop == Value.TRIP_SCORE && flag != Flag.COPY && scoreBoard.isInJam() && (Integer)value > 0) {
```

Comment:
I'd move this `if` inside the outer of the two nested`if`s above and just test for `isInJam`.

Reply:
Makes sense to me - I'll rebase all these commits into some kind of coherent structure once we have everything sorted.

Reply:
I just realized that the noop `set`s are still in.